client.go

```diff
@@ -31,7 +31,7 @@ type clientCodec struct {
 	responses map[uint64]*http.Response
 	mutex     sync.Mutex
 
-	response *Response
+	response Response
```

Comment:
What is a reason for this change?

Reply:
@kolo `Response` is just a slice of bytes so I removed the unnecessary wrapping `struct`. Since slices are already reference types, there's no need to make it a pointer.

Reply:
I can split these up into separate PRs if you want.